### PR TITLE
Training parameters (lr and weight decay) change. 

### PR DESCRIPTION
**Pull request by wuyuebupt**
Change the lr schedule from steplr to multistep lr following the paper. Change weight decay according to the incremental stages. Early stages have larger weight decay than late stages. I got one close result [0.856, 0.72125, 0.6556666666666666, 0.6015, 0.5577], see the log file in the folder logs. 

The distillation part is also different from our implementation. But I think your implementation is kind of fine.

### CODE REVIEW

**Comment by EdenBelouadah** [comment]
Applying these modifications on the LwF backbone of this code indeed improved the results. I agree with you about pulling them into the code. Thank you

**Comment by wuyuebupt** [comment]
It looks that the author doesn't maintain this repo anymore :(

**Comment by bwolfson97** [comment]
Hi @wuyuebupt, could you answer this [question](https://github.com/sairin1202/BIC/issues/5) about how many bias correction layers should be learned? It would be very much appreciated. Thanks!
